### PR TITLE
Feature splash page

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

Made a splash page
Fixed the Page Not Found bug that was displaying at the bottom of the page
Added a temporary logo

## Closes issue(s)

If there any issue that is fixed with this PR, please attach it/them here...

## Screenshots

layoutniwa

## Changes include

- [X] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [ ] I have tested this code

### CODE REVIEW

File: src/pages/Home/index.tsx [hunk 1]
```diff
@@ -15,9 +18,13 @@ export const Home = () => {
...
-      <h2>Home</h2>
-      {!userData && <SignIn />}
+    <div className="splash-page">
+      <div className="ni-wa-container">
+        <Niwa className="ni-wa-character" />
+        <h1 className="ni-wa">ni•wa</h1>
+        <h3 className="ni-wa-definition">a garden or courtyard</h3>
+        {!userData && <SignIn />}
+      </div>
     </div>
```
Comment (daiHash) [inline src/pages/Home/index.tsx:28]: Its ok to do it in a different PR,
I think we should show in the home page:
- show an explanation of how to get started and whats my niwa also when the user is not logged in 
- and show the list of gardens and a add new garden button when user is logged in

**Review by daiHash — APPROVED** [review]
Thx for the logo fix,
I think we need to decide the color scheme and try to use as much as we can components from the css framework to keep more consistency in the layout.

we can discuss it tonight , feel free to merge this PR👍

**Review by conalli — APPROVED** [review]
Yeah I agree with Daichi, we should discuss this tonight but you can merge it for now.

File: src/pages/Home/index.tsx (hunk 1 above)
Comment (curtisahuang) [reply src/pages/Home/index.tsx:28]: Sounds good!
